subiquity/ui/views/help.py

```diff
@@ -394,8 +394,9 @@ def _close(self, sender):
 
 class HelpMenu(PopUpLauncher):
 
-    def __init__(self, app):
+    def __init__(self, app, about_msg=None):
```

Comment:
This arg isn't used, correct?

Reply:
Indeed. It could be, though, but I avoided changing the SubiquityClient init code.

Reply:
Please feel free to modify the subiquity code if it makes sense to do so!